src/test/app/ValidatorList_test.cpp

```diff
@@ -564,11 +577,19 @@ class ValidatorList_test : public beast::unit_test::suite
 
             {
                 // Quorum should be 80% with all listed validators active
-                hash_set<PublicKey> activeValidators;
+                hash_set<NodeID> activeValidatorsNew{activeValidators};
                 for (auto const valKey : cfgKeys)
-                    activeValidators.emplace (*parseBase58<PublicKey>(
-                        TokenType::TOKEN_NODE_PUBLIC, valKey));
-                trustedKeys->onConsensusStart (activeValidators);
+                {
+                    auto const ins = activeValidatorsNew.emplace(
+                        calcNodeID(*parseBase58<PublicKey>(
+                            TokenType::TOKEN_NODE_PUBLIC, valKey)));
+                    if(ins.second)
+                        initiallyAdded.insert(*ins.first);
```

Comment:
I got confused by `initiallyAdded`. It seems to actually represent the second/subsequent set of added trusted keys.